bindings/java/src/org/sleuthkit/datamodel/CommunicationsManager.java

```diff
@@ -701,57 +699,62 @@ private void addAccountsRelationship(long account1_id, long account2_id, long re
 		try {
 			s = connection.createStatement();
 
-			// set up applicable filters
-			Set<String> applicableFilters1 = new HashSet<String>();
-			applicableFilters1.add(DateRangeFilter.class.getName());
-
-			String innerQuery1 = "		SELECT DISTINCT account1_id FROM relationships  AS relationships"
-					+ "		  WHERE relationships.relationship_type IN ( " + COMMUNICATION_ARTIFACT_TYPE_IDS_CSV_STR + " )";
-
-			String innerQuery2 = "		SELECT DISTINCT account2_id FROM relationships AS relationships"
-					+ "		  WHERE relationships.relationship_type IN ( " + COMMUNICATION_ARTIFACT_TYPE_IDS_CSV_STR + " )";
-
-			// Date range filters are applied to the two inner queries
-			String innerQueryfilterSQL = getCommunicationsFilterSQL(filter, applicableFilters1);
-			if (!innerQueryfilterSQL.isEmpty()) {
-				innerQuery1 += " AND " + innerQueryfilterSQL;
-			}
-			if (!innerQueryfilterSQL.isEmpty()) {
-				innerQuery2 += " AND " + innerQueryfilterSQL;
-			}
+			//TODO: this could be static 
```

Comment:
I'm not going to hold up integration for this, but @millmanorama, please either act on the TODO, remove the comment, or make a story and reference it. Otherwise, the TODO will likely live on forever....thanks!